main.go

```diff
@@ -202,28 +276,28 @@ func main() {
 	)
 
 	if err := g.Run(); err != nil {
-		logger.Fatal(err.Error())
+		logger.Fatal(errors.Wrap(err, "could not start HTTP servers").Error())
```

Comment:
If you're going to wrap it, then you can just pass the error and don't need to do `Error()`. The logger will do that for you.